cli/command/container/opts_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"time"
 
 	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/strslice"
```

Comment:
Oh! Linter is complaining; probably because the imports aren't sorted alphabetically; looks like the new import should go after the `"github.com/docker/docker/api/types/network"` import.